frontend/src/components/styles/layout/_footer.scss

```diff
@@ -1,68 +1,168 @@
+.sign {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $dark-grey;
+  width: 100%;
+  padding: 0.5rem 4rem;
+  display: flex;
+  position: fixed;
+  bottom: 0;
+
+  &__element {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 1rem;
+
+    &-login {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+
+      .btn-login {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        border: 0px;
+        width: 200px;
+        height: 45px;
+        padding: 1rem;
+        background: $dark-grey;
+        color: $white;
+        text-transform: uppercase;
+        cursor: pointer;
+        text-decoration: underline;
+        // transition: all 0.3s ease-in-out;
+      }
+
+      .btn-login:hover {
+        background: #5a5656;
+        border-radius: 20px;
+      }
+    }
+    &-signup {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+
+      .btn-signup {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 1rem;
+        background: $dark-grey;
+        color: $white;
+        border: 3px solid $orange;
+        width: 200px;
+        height: 45px;
+        box-sizing: border-box;
+        text-transform: uppercase;
+        border-radius: 20px;
+        cursor: pointer;
+      }
+
+      .btn-signup:hover {
+        background: $orange;
+        color: $white;
+      }
+    }
+  }
+
+  &__element li {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+  }
+
+  button {
+    display: flex;
+    align-items: center;
+  }
+}
+
 .footer {
   background: $dark-grey;
   width: 100%;
-  padding: 0em 4em;
+  padding: 0.5rem 4rem;
   display: flex;
   flex-direction: row;
-  // position: fixed;
+  border-top-left-radius: 20px;
```

Comment:
why did you add this border-radius? it looks weirder now.

Reply:
it's part of the Figma design, but we can change it